client/src/components/v2/OverlayMessage.tsx

```diff
@@ -39,7 +41,7 @@ export default function OverlayMessage() {
             value={message}
           />
           <InputGroup.Append>
-            <Button variant="outline-secondary" onClick={sendMessage}>
+            <Button variant="outline-secondary" onClick={() => sendMessage(message)}>
```

Comment:
This button does not clear the message text whereas pressing enter clears the text. Is there a reason for this difference? If not we should do something similar to this.
``` 
  const handleKeyPressed = useCallback(
    (event) => {
      if (event.key === 'Enter') {
        sendMessage(message);
        setMessage('');
      }
    }
```

Reply:
Done
35a4e17